warehouse/templates/packaging/detail.html

```diff
@@ -161,7 +161,7 @@ <h3 class="sidebar-section__title">GitHub statistics</h3>
           </a>
           <a class="vertical-tabs__tab vertical-tabs__tab--with-icon vertical-tabs__tab--condensed github-repo-info__item" data-key="html_url" data-attr="href" data-supplement="/issues" rel="noopener" target="_blank">
             <i class="fa fa-exclamation-circle" aria-hidden="true"></i>
-            <strong>Open issues: </strong>
+            <strong>Open Issues/PRs: </strong>
```

Comment:
This is a tremendous nitpick, but per our style guide (which is just the Material Design style guide), we'll want to use sentence-style capitalization here, so:

> Open issues/PRs

(https://material.io/guidelines/style/writing.html#writing-capitalization-punctuation)

Reply:
Do you prefer separate commits or squashes for such minor changes?

Reply:
It's a separate commit for you to do with as you choose. Screenshots updated to suit.

Reply:
> Do you prefer separate commits or squashes for such minor changes?

Separate commits are preferred because it's easier to track what's been changed, and because we squash all PRs down to a single commit when merging anyways. Thanks for asking!